documentation/Database-Schema-Template.md

```diff
@@ -33,37 +33,42 @@ The `version_id` is the link with the template version.
 
 The `template_name` and `code` is to help admin users identifying one template.
 
-The `current_status` can be `'Draft'`, `'Available'` or `'Disabled'`. The only 'Available' templates are the onesthe in the version flagged as the current one. All other application template are 'Disabled' or 'Draft' (if unfinalised).
+**To be re-considered after issue #35 is in PR or merged to master:**
+~~The `current_status` can be `'Draft'`, `'Available'` or `'Disabled'`. The only 'Available' templates are the ones in the version flagged as the current one. All other application template are 'Disabled' or 'Draft' (if unfinalised).~~
 
 ### template stage
 
 There is one or more stages per application template. Each one is defined as a new record that point to `template_id` with the `number` of this stage (1, 2, 3, ...) and title can be anything, but typically would be: `'Screening'`, `'Assessment'`, `'Final Stage'`.
 
 ### template sections
 
-Sections of the application template that contain some elements. Each section can be associated with a **template permission** if this section requires a certain type of reviewer to check for responses from an Applicant.
+Sections of the application template that contain some [elements/questions](Elements-Questions.md). Each section can be associated with a **template permission** if this section requires a certain type of reviewer to check for responses from an Applicant.
 
 ### template elements
 
 Elements in the application template always are part of a section, therefor it stores the `section_id` instead of the `template_id`.
 
-The `code` is associated with the **element type plugin**, where the definitions of each element will be coming from.
+The `code` is unique for each element in the same **template**. This code will be auto-generated, but we would allow the Admin user to change if needed.
 
-The order each element is displayed is defined by the `next_element_code` (combined with the `section_id` of the current template version).
+The `next_element_code` links to the unique code of the next element to display in the application. So we order elements as a linked list.
+
+The `title` that will be display in this question/information element to the user. Default is `'Undefined'`.
```

Comment:
Hmm, not sure about this. I thought the main display element would be `label`, which would be part of the `parameters` object (since not every element type would need one, eg. "Image"). 

If it is a display element, it should really be called `label` rather than `title`, as it will be rendered as an HTML form element label.

To be honest I just thought `title` would be an optional admin-only description of what the question is for (more detailed than `code`). Maybe we don't need it at all for now?

Reply:
Oh! Yes. I guess we had different views on that one. I agree that since now element takes care of both the `label` should be part of the parameters. I'll add the description that you had in mind to it. I think it's good to have something to describe the element like you said.